### ISSUE TITLE
auto_power_on

### ISSUE THREAD

**Issue by 3dlabsio** [PiSugar/pisugar-power-manager-rs#18]
Does this new feature allow us to not have to press the "activate" button if the battery is disconnected or loses all charge?

**Comment by fengyc** [comment]
No, it didn't. "activate" button and `auto_power_on` are different funtions.

In order to prevent the battery from being damaged,  the chip (on PiSugar2 Pro) will stop the power output when the battery loses all charge. We need to press the "activate" button to reenable it.

**Comment by 3dlabsio** [comment]
Is there a connector on the board to place the activate button outside of a case?

**Comment by cariacou** [comment]
I am having the same issue. The Pi is in a box, with no access to the "activate" button. 

Is there any way to use the PI Sugar, without using the buttons, by simply just plugging in / unplugging the USB-C cable?

**Comment by nzola** [comment]
Pisugar when plugged on power adapter turns on, but pressing power buttons on Pisugar board does nothing. It was working before. I could turn it on and off without any problems, because I know the on and off sequences.  
After the battery is fully charged, I unplug the power adaptor from Pisugar, it still on for many hours with the supply from the battery. The on and off switch still doesn't work.
